driver-core/src/main/java/com/datastax/driver/core/Configuration.java

```diff
@@ -52,7 +53,9 @@ public Configuration() {
              new SocketOptions(),
              new MetricsOptions(),
              new QueryOptions(),
-             NettyOptions.DEFAULT_INSTANCE);
+             NettyOptions.DEFAULT_INSTANCE,
+             new CodecRegistry()
```

Comment:
ahh I guess a problem could be that someone could add codecs to this which might change some default behavior down the line.